README.md

```diff
@@ -57,6 +57,11 @@ contain the name of the table. Enabling `iceberg.tables.upsert-mode-enabled` wil
 preceded by an equality delete. Both CDC and upsert mode require an Iceberg V2 table with identity fields
 defined.
 
+It is safe to reuse the same `iceberg.control.topic` for multiple connectors as this is expected to be a 
+relatively low traffic topic. Connectors can distinguish between which messages are relevant to them by looking at 
+the `iceberg.control.group-id` that is encoded inside each message for this purpose. Naturally, this does mean that 
+`iceberg.control.group-id` cannot be shared between multiple connectors. 
```

Comment:
From my reading of the code, it should be fine. Each connector will read only messages written by it's own tasks and ignore messages written by other connectors by looking at the `iceberg.control.group-id` encoded inside each message and comparing that with the `iceberg.control.group-id` the connector was configured with [here](https://github.com/tabular-io/iceberg-kafka-connect/blob/4c4b8f8408846926545ae453cfe4ffbdae7801cd/kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Channel.java#L127).

Although it's not called out anywhere explicitly in the docs, my understanding is that it's probably even expected to reuse the same control topic for multiple connectors.

Now if someone was to run thousands of iceberg sink connectors using the same control topic, then potentially there might be some performance concerns. But that should be quite far from the common case and in that case they can always have multiple control topics and split connectors between them. 

Would appreciate a confirmation that my assumptions above are correct.

Reply:
@bryanck would appreciate a sanity check of the above :)